client-app/src/Components/NewItemForm.tsx

```diff
@@ -183,51 +278,73 @@ const NewItemForm: React.FC = () => {
     const handleRefresh = () => {
         setFormData({
             itemType: '',
-            currentStatus: 'Received',
+            currentStatus: '',
```

Comment:
Revert this, current status should always be defaulted to 'Received' when item is first created